src/arrayMethodIndexOf.js

```diff
@@ -6,7 +6,19 @@
  */
 function applyCustomIndexOf() {
   [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+    // return searchElement === undefined ? -1
+    //   : this.slice(fromIndex).findIndex(el => Object.is(el, searchElement));
+    if (searchElement === undefined) {
+      return -1;
+    } else {
+      let arr = this.slice(fromIndex);
+      for (let i = 0; i < arr.length; i++) {
```

Comment:
You don't need to slice an array. You can just write `for (let i = fromIndex; ...)`

Reply:
Also, it returns the wrong index. We'll update the test case.

```
const array = [0, 1, 2, 3, 4, 5, 6];
const fromIndex = 4;
const searchElement = 4;

const sliced = array.slice(fromIndex); // [4, 5, 6]
```
When you iterate over `sliced` the `index` will be `0` according to your solution. But really it is 4.